README.md

```diff
@@ -72,3 +72,16 @@ mvn -Dkotlin.version=1.4.10 -DskipTests package
 To start the shell, run `bin/ki.sh` on Linux and macOS. On Windows, use `bin\ki.bat` instead.
 
 To exit the shell, type `:q` or `:quit`.
+
+## Adding maven repositories that require auth
+
+The following options are supported:
+
+```bash
+:repository https://myrepo.org username password
```

Comment:
This format will make it difficult to use other auth schemes, e.g. keyfile/passphrase.
I'd suggest to prefix user/pw with keys or use the same format as in properties file. e.g.
```
:repository https://myrepo.org username:john password:johnpwd
```
or
```
:repository https://myrepo.org username=john password=johnpwd
```

Reply:
+

Reply:
All good now, thanks!